lib/detail_rich_text.dart

```diff
@@ -0,0 +1,41 @@
+import 'ui_constants.dart';
+import 'package:flutter/material.dart';
+
+class DetailRichText extends StatelessWidget {
+  const DetailRichText({
+    super.key,
+    required this.subtitle,
+    required this.text,
+  });
+
+  final String subtitle;
+  final String text;
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: MediaQuery.of(context).size.width,
```

Comment:
I think this parameter is not needed. If not specified, `SizedBox` adjusts to the size it has available from the parent. Give it a try, I think it'll work!

Reply:
You were right ;)
(actually, I removed the SizedBox altogether, not only the width parameter -I don't remember why it was there in the first place).